code/modules/projectiles/guns/oddity_items.dm

```diff
@@ -424,6 +426,36 @@
 	to_chat(user, SPAN_NOTICE("You turn the [src] off."))
 	..()
 
+/obj/item/tool/sword/katana/crimson_arc
+	name = "\"Crimson Arc\" katana"
+	desc = "An anomalous weapon created by rivals of the unknown person(or group?) of the bluecross, their work marked by a crimson cross, these weapons are known to vanish and reappear when left alone. \
+			What seems to be a normal katana but with a red hilt but with a nasty trick, the closer the user is to death the faster it becomes to swing."
```

Comment:
```suggestion
			What seems to be a normal katana but with a red hilt and a nasty trick, the closer the user is to death the faster it becomes to swing."
```

Not a spelling issue but using 2 buts like this is odd. Nit-picking but regardless.